services/ir/.ir.env

```diff
@@ -1,6 +1,20 @@
 NEOFS_IR_LOGGER_LEVEL=debug
-NEOFS_IR_METRICS_ENABLED=true
-NEOFS_IR_PPROF_ENABLED=true
+
+# Profiler section
+## Server address. Empty value disables profiler
+## Default: <empty> (invalid)
```

Comment:
```suggestion
## Default: <empty>
```
For metrics too.

Reply:
Means that empty value is invalid

Reply:
The line above says that empty value disables profiler, so I support is it valid, isn't it?

Reply:
Ok, it's valid in program, but invalid way to serve profiler. I'll remove `invalid`

Reply:
Maybe `## Default: <empty> (disabled)` will be better.